backend/utils.py

```diff
@@ -1,5 +1,15 @@
 import aiofiles
 import urllib
+import uuid
+try: 
```

Comment:
@Ayman-Ameen can you explain this? We've already added try catch for importing this libraries within the function calls. Did you merge latest master?